File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = ''
    for word in s.split(" "):
        for i in range(len(word)):
            if i % 2 == 0:
                answer += word[i].upper()
            else:
                answer += word[i].lower()
                
        answer += " "    
    
    return answer[:len(answer)-1]

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String[] arr = s.split("");
        int loc = 0;
        
        for (String i: arr){
            loc = (i.equals(" ")) ? 0 : loc + 1;
            i = (loc % 2 == 0) ? i.toLowerCase() : i.toUpperCase();
            answer += i;
        }
        
        return answer;
    }
}